fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfAttachment.java

```diff
@@ -136,6 +136,9 @@ public static String generate(CadfAttachment obj)
 
         public static void generate(CadfAttachment obj, JsonGenerator generator)
```

Comment:
maybe some javadoc to explain the behavior when null is passed?

Reply:
done